### ISSUE TITLE
Hazelnut block is obtainable

### ISSUE THREAD

**Issue by KAfable** [MatrexsVigil/harvestcraft#310]
![image](https://user-images.githubusercontent.com/26587049/50728364-77d5a480-10f6-11e9-9cbf-b6a7313d7740.png)

Hazelnut block is obtainable, all other fruit trees only drop the fruit (eg Persimminon), otherwise players can use the Hazelnut sprout to make as many as they need.

**Comment by Draskh** [comment]
This is indeed the case. 
Version: zb
Forge: 2796

**Comment by HeroOfChaos** [comment]
This actually seems to work with all of the more recently added fruit trees, like Pawpaw or Lychee.

**Comment by luke-bravenboer** [comment]
Huh, seems like it works on all fruit/nut trees actually (with shears- were you all using shears too?)

**Comment by Draskh** [comment]
No, I didn't use shears. I think shears are the intended way to obtain the blocks.

**Comment by Danymaddox** [comment]
This bug works with all of these fruits:
Hazelnut
Tamarind
Passionfruit
Jackfruit
Rambutan
Pawpaw
Breadfruit
Guava
Lychee
Soursop

Forge : 14.23.5.2815
Harvestcraft version: 1.12.2zb

**Comment by Danymaddox** [comment]
Also, the block can be insta mined when fully gown

**Comment by Nitro-Carwash** [comment]
This is an issue with Hunger Overhaul.  I have a PR to fix this here: https://github.com/progwml6/HungerOverhaul/pull/240
